install-moveit2/source/index.markdown

```diff
@@ -13,7 +13,9 @@ Installing MoveIt 2 from source is the first step in contributing new features,
 
 MoveIt is mainly supported on Linux, and the following build instructions support in particular:
 
-- Ubuntu 20.04 / ROS 2 Foxy Fitzroy
+- Ubuntu 20.04 / ROS 2 Foxy Fitzroy (LTS)
+- Ubuntu 20.04 / ROS 2 Galactic Geochelone (Recommended)
+- Ubuntu 20.04 / ROS 2 Rolling Ridley (Development)
```

Comment:
```suggestion
- Ubuntu 20.04 / ROS 2 Rolling Ridley (Continuously Updated)
```
This seems to be the word [they use](https://docs.ros.org/en/foxy/Releases/Release-Rolling-Ridley.html), and I find it more descriptive.